pages/_app.js

```diff
@@ -39,6 +42,11 @@ function MyApp({ Component, pageProps, router }) {
       />
       <Header />
       <Component {...pageProps} />
+      {isDemoRoute && (
+        <GithubCodeLink
```

Comment:
This won't be a good place to add the github button. think about a demo that takes a lots of scroll... so the github demo button will be at the bottomof the page. That will be hard for users to find.

Here is an example: https://webapis-playground-git-fork-aanchal-fatwani-bca5fa-thoughtdump.vercel.app/demos/picture-in-picture

Reply:
Acknowledged the issue. Regd. the same would like to understand what you have in mind for better placement.

1. Placing the link after the header would act redundantly for the user as the header has a similar link.
2. Placing the link after the content would cause problems where we have a large section for content.

As a proposition, having a floating button might be a good option.
Here are some of the ideas as floating on the left side.
(RHS for floating icons is generally used for traversal aid such as 'Go To Top' features, which might be a future addition to the appln., if need be)
### Suggestion 1

#### First Fold:
![SeeCode1](https://user-images.githubusercontent.com/63228776/135713932-9c0f7f0f-4a9c-4c52-9aa3-1d899419dc27.PNG) 
#### Scrolled View:
![SeeCode2](https://user-images.githubusercontent.com/63228776/135713924-20d4cca8-74a7-4fc8-8245-026294e8bd4a.PNG)

### Suggestion 2 - Only Icon
#### First Fold:
![GitLink1](https://user-images.githubusercontent.com/63228776/135713977-144e30ec-de53-41e6-9828-667434b364a3.PNG)
#### Scrolled View:
![GitLink2](https://user-images.githubusercontent.com/63228776/135713983-390d86de-e7ce-4c8f-aeff-7210cf3ed97b.PNG)

@atapas 
**Please suggest which idea suits your requirement, or any place specific you have in mind.**

Reply:
May be keep the floating button on the right side

Reply:
Yep I liked that feedback.. it will be great usability feedback. @aanchal-fatwan